pkg/common-controller/framework_test.go

```diff
@@ -164,16 +164,17 @@ type reactorError struct {
 	error    error
 }
 
-func withSnapshotFinalizer(snapshot *crdv1.VolumeSnapshot) *crdv1.VolumeSnapshot {
-	snapshot.ObjectMeta.Finalizers = append(snapshot.ObjectMeta.Finalizers, utils.VolumeSnapshotContentFinalizer)
```

Comment:
Ok, I see that utils.VolumeSnapshotContentFinalizer is added to VolumeSnapshot API object here.  This is wrong.  We only need utils.VolumeSnapshotContentFinalizer for VolumeSnapshotContent, not for VolumeSnapshot.

Reply:
ok, removed the contentFinalizer.